exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -34,46 +36,42 @@ import (
 func (c *CoinbasePro) SetDefaults() {
 	c.Name = "CoinbasePro"
 	c.Enabled = true
-	c.Verbose = true
 	c.API.CredentialsValidator.RequiresKey = true
 	c.API.CredentialsValidator.RequiresSecret = true
-	c.API.CredentialsValidator.RequiresClientID = true
-	c.API.CredentialsValidator.RequiresBase64DecodeSecret = true
-
+	c.API.CredentialsValidator.RequiresBase64DecodeSecret = false
 	requestFmt := &currency.PairFormat{Delimiter: currency.DashDelimiter, Uppercase: true}
 	configFmt := &currency.PairFormat{Delimiter: currency.DashDelimiter, Uppercase: true}
-	err := c.SetGlobalPairsManager(requestFmt, configFmt, asset.Spot)
```

Comment:
My config is not being updated to support futures, despite valid credentials.
Also, I would ask that you seperate futures into `asset.DeliveryFutures` for long dated futures and `asset.Futures`/`asset.PerpetualContract`/`asset.PerpetualSwap`